modules/samples/oauth2/playground2/src/main/webapp/oauth2.jsp

```diff
@@ -47,6 +49,7 @@
 
         error = request.getParameter(OAuth2Constants.ERROR);
         grantType = (String) session.getAttribute(OAuth2Constants.OAUTH2_GRANT_TYPE);
+        implicit_response_type = (String) session.getAttribute(OpenIDConnectConstants.IMPLICIT_RESPONSE_TYPE);
```

Comment:
Use camel case for variable names
ie. implicit_response_type --> implicitResponseType